### PR TITLE
#170421866 -  view high rated accommodations

### PR DESCRIPTION
**Pull request by sabin18**
#### What does this PR do?
PR  to view high rated accommodations
#### Description of Task to be completed?
- add view high rated components
- add view high rated action
#### How should this be manually tested?
- Clone ft-notifications-pane-168781628 branch
- run `npm i`
- run `npm run dev`
- after login, you will navigate to the home page where you will see high rate accommodations
#### Any background context you want to provide?
`N/A`
#### What are the relevant pivotal tracker stories?
[#170421866](https://www.pivotaltracker.com/story/show/170421866)
#### Screenshots (if appropriate)
Screen Shot 2019-12-30 at 13 41 37

#### Questions:
`N/A`

### CODE REVIEW

**Review by alainmateso — APPROVED** [review]
LGTM

**Review by Cheza-Dzabala — APPROVED** [review]
Awesome job

File: src/components/pages/landingPage/HighRatedAccommodations.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+/* eslint-disable react/prefer-stateless-function */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { withRouter, Link } from 'react-router-dom';
+
```
Comment (nakiwuge) [inline src/components/pages/landingPage/HighRatedAccommodations.js]: Remove this line

File: src/components/pages/landingPage/HighRatedAccommodations.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    async componentDidMount() {
+      const { getHighRatedAccommodation } = this.props;
+      this.setState({ isLoading: true });
+      await getHighRatedAccommodation();
+      this.setState({ isLoading: false });
+    }
+
+    render() {
+      const { isLoading } = this.state;
+      const { highRated } = this.props;
```
Comment (nakiwuge) [inline src/components/pages/landingPage/HighRatedAccommodations.js:25]: Add a new line

File: src/components/pages/landingPage/HighRatedAccommodations.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+import { Row, Col } from 'react-bootstrap';
+import { getHighRatedAccommodation } from '../../../actions/accommodationActions';
+
+export class HighRatedAccommodation extends Component {
+    state= {
+      isLoading: false,
+    }
+
+    async componentDidMount() {
+      const { getHighRatedAccommodation } = this.props;
```
Comment (nakiwuge) [inline src/components/pages/landingPage/HighRatedAccommodations.js:16]: Add a new line

**Review by nakiwuge — CHANGES_REQUESTED** [review]
Good job @sabin18. Kindly work on the feedback below.
- Make the `r ` in the heading `High rated` Capital.
- As shown in the screen shot below the high rated images are not alligned with `All destinations`. Ensure that the image sizes are aligned when the screen size reduces.
- The rating also moves from the blue background when in mobile view.
- I have left other comments on the pr




Screenshot_20191230-225744

**Review by nakiwuge — APPROVED** [review]

**Pull request merged**